oga/frontend/src/containers/Answer/NewAnswer.js

```diff
@@ -0,0 +1,131 @@
+import React, { Component } from 'react';
+import './NewAnswer.css';
+
+import { connect } from 'react-redux';
+import * as actionCreators from '../../store/actions/';
+
+import { Redirect } from 'react-router-dom';
+import { push } from 'connected-react-router';
+import Map from '../Map/GoogleMap';
+import PushNotification from '../../components/PushNotification/PushNotification'
+import NewQuestion from '../QuestionList/NewQuestion/NewQuestion';
+
+class NewAnswer extends Component {
+  state = {
+    answer_content: '....',
+  }
+
+  componentDidMount() {
+    this.props.onGetQuestion(this.props.match.params.id);
+  }
+
+  postQuestionHandler = () => {
```

Comment:
are we posting question here? or answer?

Reply:
I forgot to change name. thanks